src/controllers/project-controller.js

```diff
@@ -159,3 +162,108 @@ export async function getDuplicationRequestCriteria(req, res) {
 
   res.status(200).json({ oneTimeMedia, projectPublished, hasAccess })
 }
+
+export async function getPartitionSummary(req, res) {
+  const projectId = req.params.projectId
+  const partitionId = req.params.partitionId
+
+  const partition = await models.Partition.findByPk(partitionId)
+  let onetimeMedia = []
+  let views = 0
+  let specimens = 0
+  let labels = 0
+
+  const characters = await partitionService.getCharactersInPartitions(
+    partitionId
+  )
+  const taxa = await partitionService.getTaxaInPartitions(partitionId)
+
+  const taxaIds = Array.from(taxa.values())
+  const characterIds = Array.from(characters.values())
+
+  const taxaMedia =
+    taxaIds.length != 0 ? await mediaService.getMediaByTaxaIds(taxaIds) : []
+  const characterMedia =
+    characterIds.length != 0
+      ? await mediaService.getMediaByCharacterIds(characterIds)
+      : []
+
+  const connectedMediaIds = taxaMedia.filter((media) => {
+    return !characterMedia.includes(media)
+  })
```

Comment:
I don't understand what it makes to be "connected". What is this suppose to represent?

Reply:
This path seems to deviate from the `getMediaSpecimensAndViews` method - which although was poorly named, is an accurate tally of this information. Can you add it here.